scripts/CEE/delete-worker-node/metadata.yaml

```
@@ -0,0 +1,30 @@
+file: script.sh
+name: delete-worker-node
+description: Delete/Replace worker nodes
+author: Nelson Paez
+allowedGroups:
+  - CEE
+  - SREP
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - get
+        - patch
+        - list
+        - delete
+      apiGroups:
+        - ''
+        - 'machine.openshift.io'
+      resources:
+        - nodes
+        - machines
```

```suggestion
  clusterRoleRules:
    - verbs:
        - get
        - list
      apiGroups:
        - ''
      resources:
        - nodes
  roles:
    - namespace: "openshift-machine-api"
       rules:
          - verbs:
            - get
            - list
            - delete
            apiGroups:
            - "machin.openshift.io"
            resources:
            - "machines"
```

The `machine` resource is namespaced, we can scope it to `openshift-machine-api`.